web/src/pages/project/ProjectGrid.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react'
+import _ from 'lodash'
+import { Table } from 'semantic-ui-react'
+
+import SampleLink from '../../shared/components/links/SampleLink'
+import FamilyLink from '../../shared/components/links/FamilyLink'
+import sanitiseValue from '../../shared/utilities/sanitiseValue'
+import { ProjectSummaryResponse } from '../../sm-api/api'
+import MuckError from '../../shared/components/MuckError'
+
+interface ProjectGridProps {
+    summary?: ProjectSummaryResponse
+    projectName?: string
+    error?: string
+}
+
+const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
+    summary,
+    projectName,
+    error,
+}) => {
+    if (!projectName) {
+        return (
+            <p>
+                <em>Please select a project</em>
+            </p>
+        )
+    }
+    if (error) {
```

Comment:
As a suggestion, I think that rendering the error and loading states should be a responsibility of the parent component that is fetching the data. Otherwise it seems like the data fetching abstraction is leaking into this child component which should only be responsible for rendering the summary data.